frontend/components/product-list/sections/FilterableProductsSection/FacetsAccordion.tsx

```diff
@@ -98,7 +98,24 @@ export const FacetAccordionItem = forwardRef<FacetAccordionItemProps, 'div'>(
       const isDisabled = isProductListEmpty || !hasApplicableRefinements;
 
       const formattedFacetName = formatFacetName(attribute);
-      const isHidden = !hasApplicableRefinements && !isProductListEmpty;
+      let isHidden = !hasApplicableRefinements && !isProductListEmpty;
+      const emptyProductList = isProductListEmpty && !hasApplicableRefinements;
+
+      let isPartsPage = productList.type === ProductListType.AllParts;
+
+      const ancestors = productList.ancestors;
+      if (ancestors[ancestors.length - 1]) {
+         const path = ancestors[ancestors.length - 1].path;
+         isPartsPage = path.includes('/Parts');
+      }
```

Comment:
Will this produce the same effect? I'm don't understand the purpose of looking at the path of the second to last ancestor, if I'm missing something lmk.

```suggestion
      const isPartsPage =
         productList.type === ProductListType.AllParts
         || productList.type === ProductListType.DeviceParts;
```

Reply:
Resolved 451fdd85a5550cd34a1f611c1383149be57d57ac